frontend/form-builder/src/app/App.tsx

```diff
@@ -1,34 +1,50 @@
 import React from "react";
 import "@digdir/design-system-tokens/brand/digdir/tokens.css";
 import classes from "./App.module.css";
+import { SubHeader } from "../../../main-app/src/components/SubHeader";
 import { Toolbar } from "../components/Toolbar";
 import { FormBoard } from "../components/FormBoard";
 import { SettingsSidebar } from "../components/SettingsSidebar";
-import { Heading } from "@digdir/design-system-react";
-import { useTranslation } from "react-i18next";
 import { DndProvider } from "react-dnd";
 import { HTML5Backend } from "react-dnd-html5-backend";
+import { Heading, Link } from "@digdir/design-system-react";
+import { FloppydiskFillIcon, TasklistSendFillIcon } from "@navikt/aksel-icons";
+import { useTranslation } from "react-i18next";
 
 export const App = (): React.JSX.Element => {
   const { t } = useTranslation();
   return (
     <>
-      <Heading level={1} spacing className={classes.headline}>
-        {t("form_builder")}
-      </Heading>
-      <div className={classes.builderGrid}>
-        <DndProvider backend={HTML5Backend}>
-          <div className={classes.builderSection}>
-            <Toolbar />
-          </div>
+      <SubHeader>
+        <Heading className={classes.subHeaderHeading} level={2} size="xxsmall">
+          Form Builder
+        </Heading>
+        <div className={classes.subHeaderLinks}>
+          <Link>
+            {t("form_builder.save")}
+            <FloppydiskFillIcon className={classes.subHeaderIcon} />
+          </Link>
+          <Link>
+            {t("form_builder.publish")}
+            <TasklistSendFillIcon className={classes.subHeaderIcon} />
+          </Link>
+        </div>
+      </SubHeader>
+      <main className={classes.main}>
```

Comment:
should we maybe use a different tag here? Since the main tag is already used on a higher lvl.
<img width="311" alt="image" src="https://github.com/ErlingHauan/FormFactory/assets/74791975/567cbc30-26c3-4382-a82c-1268c6e9bed8">